### PR TITLE
Updated queries for adminApproved

### LINKED ISSUE #1693: [userTypeFix] Update queries for new `adminApproved` functionality
**Is your feature request related to a problem? Please describe.**
Changes to queries have to be made in Admin repo corresponding to https://github.com/PalisadoesFoundation/talawa-api/issues/1862

**Describe the solution you'd like**
Change GraphQl Queries

**Describe alternatives you've considered**

**Approach to be followed (optional)**
Make changes only in the "develop-userType" branch

**Additional context**
follow this issue's description -> https://github.com/PalisadoesFoundation/talawa-api/issues/1862

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Rajeshpatel07**
**What kind of change does this PR introduce?**
 A feature

**Issue Number:**

Fixes #1693 

**Did you add tests for your changes?**
 Yes

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

In the following Pull Request, I have made updates to the queries related to adminApproved to ensure correct functionality.

**Does this PR introduce a breaking change?**
 No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
 Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
1. `adminApproved` needs to be on a per organization basis. You'll need to change the schema.
2. You'll also need to write the tests to verify your work. The patch code coverage is too low (75%)

    image

File: schema.graphql [hunk 1]
```diff
@@ -94,6 +94,7 @@ scalar Any
 type AppUserProfile {
   _id: ID!
   adminFor: [Organization]
+  adminApproved: Boolean!
```
Comment (palisadoes) [inline schema.graphql:97]: `adminApproved` needs to be on a per organization basis. You'll need to change the schema to be similar to that of `adminFor`

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: schema.graphql (hunk 1 above)
Comment (Rajeshpatel07) [reply schema.graphql:97]: Okay I will make that change for `adminApproved` to be like `adminFor`.

**Comment by Rajeshpatel07** [comment]
As per the `codecov/patch` I have a confussion.  Should I add tests for `memberDetails.tsx` or `Home.tsx` .
 when i see the the full [View full report in Codecov by Sentry ](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin/pull/1756?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=PalisadoesFoundation)  it has `Home.tsx` with `26.51%` Uncovered. So can you please clarify this to me.
Thank you.

**Comment by palisadoes** [comment]
This is an update on the PR merging freeze:

1. In the next few hours we will be merging the `develop-userTypeFix` branch into the `develop` branch. 
1. The `develop-userTypeFix` branch was created to fix a long standing design flaw where Admins were Admins of all organizations, not specific ones.
1. The `userType` field has been removed from the User collection and it has been replaced by an `appUserProfileId` field.
    1. This field is `null` if the user isn’t registered to use the apps. This will help people to add users manually during the event checkin process, or if an Admin wants to manually add someone in the Admin dashboard.
    1. When not `null` the `AppUserProfileID` will reference a `AppUserProfile` collection with App related information such as the organizations for which a user may be an Admin.
    1. The updated schema can be found here https://github.com/PalisadoesFoundation/talawa-api/blob/develop-userTypeFix/schema.graphql
    1. This is the parent issue that we have been using to track progress:
        1. https://github.com/PalisadoesFoundation/talawa-api/issues/1965
1. This merge will cause some conflicts in your PR.

We decided to do this at the beginning of the weekend to give us all time to adjust PR code and create bug fixes that may arise.

Update your code at or after midnight GMT on the morning of March 23, 2024. (5:30am IST). 

If your PRs have already been approved, request a re-review after fixing the conflicts and refactoring to the new `AppUserProfileID` methodology.

**Comment by palisadoes** [comment]
Thanks for submitting your PR, but we must close it.

1. We get notified of every event in the queue, and draft PRs like these, and their updates that require no action can be a distraction. We need our reviewers to focus on updates that are ready. 
1.  Don't be discouraged, we are just trying to keep our workloads to a manageable level.

Please resubmit when you are ready.

**Pull request closed without merging**